sfepy/examples/linear_elasticity/nodal_lcbcs.py

```diff
@@ -47,15 +47,11 @@
 
 View the 2D results using::
 
-  python postproc.py square_quad.vtk --wireframe -b
-
-  python postproc.py square_quad.vtk --wireframe -b --only-names=u -d'u,plot_displacements,rel_scaling=1,color_kind="scalars",color_name="von_mises_stress"'
+  python resview.py square_quad.vtk -2
```

Comment:
Also show deformed mesh.

Reply:
No deformed mesh is promised in the text, so the deformations are indicated by the glyphs.